examples/provide-liquidity/components/wallet/Wallet.tsx

```diff
@@ -107,6 +135,29 @@ export const WalletSection = () => {
     />
   );
 
+  const Balance = () => {
+    return (
+      <HStack w="256px" py={2} fontSize="13px" justify="space-between">
+        <Flex gap={1}>
+          <Text fontWeight="semibold">
+            Balance: {osmoBalance?.amount || '--'}
+          </Text>
+          <Text pt="1px" fontSize="12px" color="blackAlpha.700">
+            {osmoBalance?.denom}
+          </Text>
+        </Flex>
+        <Flex gap={1}>
+          <Text fontWeight="semibold">
+            Staked: {balanceStaked?.amount || '--'}
```

Comment:
actually this will be incorrect since it's possible that `.amount` = 0 which would show `--`